doc/GUIDE.md

```diff
@@ -267,8 +193,8 @@ test/Spec.hs
 The `app/Main.hs`, `src/Lib.hs`, and `test/Spec.hs` files are all Haskell source
 files that compose the actual functionality of our project (we won't dwell on
 them here). The `LICENSE` file has no impact on the build, but is there for
-informational/legal purposes only. The files of interest here are `Setup.hs`,
-`helloworld.cabal`, and `stack.yaml`.
+informational/legal purposes only, as well as the `README.md`. The files of interest here
```

Comment:
This is a little awkwardly worded. Maybe something like "The `LICENSE` and `README.md` files have no impact on [...]"